zmon-agent.py

```diff
@@ -60,11 +59,12 @@ def get_hash(ip):
 def get_tags_dict(tags):
     return { t['Key']: t['Value'] for t in tags }
 
-def assign_stack_name_and_version_from_tags(obj, tags):
-    if 'StackName' in tags:
-        obj['stack_name'] = tags['StackName']
-    if 'StackVersion' in tags:
-        obj['stack_version'] = tags['StackVersion']
+def assign_properties_from_tags(obj, tags):
+    import inflection
```

Comment:
Maybe we need to check that there are no collisions? What about tags like "id" etc possibly overwriting other fields.